### DISCUSSION TITLE
Welcome to iModel.js Discussions!

### DISCUSSION THREAD

**Discussion by calebmshafer** [iTwin/itwinjs-core#90] [General] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with the members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Ask for advice about using iModel.js  
  * Share ideas.
  * Engage with other community members

**Comment by jan-slegr** [comment]
Hi Caleb,

I am confused a bit where to ask for what. Is this split, what should be used for what, correct?

* [GitHub issues](https://github.com/imodeljs/imodeljs/issues) to report bugs found in imodeljs code and API documentation
* StackOverflow to ask how to use API (with [imodeljs tag used](https://stackoverflow.com/questions/tagged/imodeljs))
* [GitHub discussions](https://github.com/imodeljs/imodeljs/discussions) for "everything else"

Regards,

  Jan

**Comment by pmconne** [comment, marked as answer]
It was always awkward to have to redirect API questions to a different site. Now that github supports discussion forums, it is much simpler:
* [GitHub issues](https://github.com/imodeljs/imodeljs/issues) to report bugs.
* [GitHub discussions](https://github.com/imodeljs/imodeljs/discussions) for "everything else".

The contributors' guide will be updated very shortly to reflect this.
